src/services/sendEmailService.ts

```diff
@@ -0,0 +1,55 @@
+const sendGridMail = require('@sendgrid/mail');
```

Comment:
I suggest sendgrid to be a sendEmail function parameter - we can then mock its behavior in tests easily